src/types.ts

```diff
@@ -0,0 +1,2 @@
+// alias type for uuid v4
+export type UUID = string;
```

Comment:
I wonder if it would make sense to write a boxing type to have additional runtime guarantees (e.g. UUIDs could be always validated at instantiation time). Not sure if it's worth the overhead though.

Reply:
This is in itself a very good suggestions, though I recall that some of our tests (in various repos) assign random strings to ids, which is fine when `UUID` is just an alias for string. Maybe we should make it less flexible for our own sake.